src/components/Errors/Errors.tsx

```diff
@@ -0,0 +1,26 @@
+import { ErrorSpec } from '../../types/ErrorSpec';
+import { ErrorMessage } from '../ErrorMessage/ErrorMessage';
+
+type Props = {
+  error: ErrorSpec | null;
+  closeError: () => void;
+};
+
+export const Errors: React.FC<Props> = ({ error, closeError }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className="notification is-danger is-light has-text-weight-normal"
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        aria-label="none"
```

Comment:
`<button>none</button>` used for what?
the value for the aria-attribute should answer the question what this button does
this button closes the error, so `aria-label="Hide error"` will be better